### ISSUE TITLE
Bug: The play() request was interrupted by a new load request

### ISSUE THREAD

**Issue by cmaerz** [video-dev/can-autoplay#48]
Version 3.0.1 give me the following error:

![image](https://user-images.githubusercontent.com/1479186/129527063-5b52654a-a42b-4510-9045-5fc4e5a2464f.png)

In Version 3.0.0 everything is fine.

My Code is the following:
```
canAutoPlay
          .video({ timeout: 4000, muted: false, inline: true })
          .then((data) => {
            console.log(data);
            const { result } = data;
            const startAutoplay = result === true;
            store.dispatch(actions.init(startAutoplay));
            finishPlayerInitialization(startAutoplay);
          });
```

**Comment by gkatsev** [comment]
Oh, I think I see the problem. I think that the change that was added https://github.com/video-dev/can-autoplay/commit/66c406e915dea5394552287878a928a05f5d9b77 is likely in the wrong place. It should probably move into `sendOutput`.

**Comment by gkatsev** [comment]
Published 3.0.2 with a fix. Can you please try it out and let me know if you still see the issue, @cmaerz?

Thanks!
